OrbitGl/App.cpp

```diff
@@ -1161,26 +1140,25 @@ void OrbitApp::OnRemoteProcessList(const Message& a_Message) {
 void OrbitApp::OnRemoteModuleDebugInfo(const Message& a_Message) {
   std::istringstream buffer(std::string(a_Message.m_Data, a_Message.m_Size));
   cereal::BinaryInputArchive inputAr(buffer);
-  std::vector<ModuleDebugInfo> remoteModuleDebugInfo;
-  inputAr(remoteModuleDebugInfo);
+  std::vector<ModuleDebugInfo> remote_module_debug_infos;
+  inputAr(remote_module_debug_infos);
 
-  for (auto& moduleInfo : remoteModuleDebugInfo) {
-    // Get module from name
-    std::string name = ToLower(moduleInfo.m_Name);
+  for (ModuleDebugInfo& module_info : remote_module_debug_infos) {
```

Comment:
the loop parameter can be made const, can't it?

Reply:
It required some other changes, but I made it happen -> Done